lkmpg.tex

```diff
@@ -1502,6 +1502,13 @@ \subsection{Flashing keyboard LEDs}
 Flashing keyboard LEDs can be such a solution: It is an immediate way to attract attention or to display a status condition.
 Keyboard LEDs are present on every hardware, they are always visible, they do not need any setup, and their use is rather simple and non-intrusive, compared to writing to a tty or a file.
 
+Since v4.14, the timer API started to change. The series of improvements ended up at v4.15.
```

Comment:
List the example which illustrates the API changes between v4.15+ and older ones.

Reply:
That is, a minimal code snip here.